src/card.css

```
@@ -0,0 +1,71 @@
+.card {
+  background-color: var(--white-color);
+  max-width: 200px;
+  border: 1px solid #f3f3f3;
```

don't forget to create variables for all colors
```suggestion
  border: 1px solid $c-color;
```